______________________________________________________________________

[24/May/2017 19:04:38] "GET /simulator/simulationrun/ HTTP/1.1" 302 0
[24/May/2017 19:04:41] "GET /simulator/ HTTP/1.1" 200 9584
Exception ignored in: <bound method Image.__del__ of <tkinter.PhotoImage object
at 0x05D04BB0>>
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36-32\lib\tkinter\__init__.py", line 3501,
in __del__
    self.tk.call('image', 'delete', self.name)
RuntimeError: main thread is not in main loop
Tcl_AsyncDelete: async handler deleted by the wrong thread

https://github.com/PyOCL/OpenCLGA/issues/23
______________________________________________________________________


______________________________________________________________________


______________________________________________________________________


______________________________________________________________________

______________________________________________________________________


______________________________________________________________________



______________________________________________________________________


______________________________________________________________________



______________________________________________________________________


______________________________________________________________________



______________________________________________________________________


______________________________________________________________________



______________________________________________________________________


______________________________________________________________________



______________________________________________________________________


______________________________________________________________________

